src/bsoncxx/json.hpp

```diff
@@ -34,16 +34,26 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 extern const bool k_silence_doxygen;
 
 ///
-/// Converts a BSON document to a JSON string.
+/// An enumeration of the types of Extended JSON that the to_extended_json function accepts
+///
+/// The values are:
+///   k_legacy: to produce Legacy Extended JSON
```

Comment:
I'll bet doxygen has a way to explicitly doc comment enumeration values, if so we should use that.